Chapter_04/python/2.0.0/com/sparksamples/movie_data.py

```diff
@@ -36,11 +36,11 @@ def main():
     fig = matplotlib.pyplot.gcf()
     fig.set_size_inches(16, 10)
 
-    for tick in f.xaxis.get_major_ticks():
-        tick.label.set_fontsize(8)
-                # specify integer or one of preset strings, e.g.
-                #tick.label.set_fontsize('x-small')
-        tick.label.set_rotation('vertical')
+    # for tick in ax.xaxis.get_major_ticks():
```

Comment:
since it is only a variable name change we might not want to change it as of now as it affects the code in the book